pegasus/helpers/analytics_constants.rb

```diff
@@ -38,6 +38,7 @@ module AnalyticsConstants
     VIDEO_LIBRARY_PAGE_VISIT_EVENT = 'Video Library Page Visited'.freeze,
     INTL_PL_PAGE_VISIT_EVENT = 'International Professional Learning Page Visited'.freeze,
     INTL_PL_PARTNERS_PAGE_VISIT_EVENT = 'International Professional Learning Partners Page Visited'.freeze,
-    MUSIC_PAGE_VISITED_EVENT = 'Music Lab Marketing Page Visited'.freeze
+    MUSIC_PAGE_VISITED_EVENT = 'Music Lab Marketing Page Visited'.freeze,
+    MUSIC_LAB_PAGE_VISITED_EVENT = 'Music Lab Tools Page Visited'.freeze
```

Comment:
Amplitude event is working:
<img width="435" alt="Screenshot 2024-05-16 at 8 48 44 AM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/e93b5378-0359-46d3-8782-014e63a0b0ad">